tartiflette/parser/visitor/visitor.py

```diff
@@ -505,7 +514,9 @@ def _validates_vars(self) -> None:
         a_value = self._vars[name]
 
         if self._internal_ctx.node.is_list:
-            if not isinstance(a_value, list):
+            if not isinstance(a_value, list) and (
+                a_value is not None or (a_value is None and not is_nullable)
```

Comment:
Shouldn't the condition be:
```suggestion
                a_value is not None
```

This way, if a `None` value is provided, we'll hit the `self._validate_type(name, a_value, a_type, is_nullable)` at line 538 which (IMO) will raise a more relevant error:

> Value can't be null or contain a null value

Reply:
Hum, let me test it, it was clear on my mind when I wrote it 20minutes ago ...

Reply:
You're right, it was a little bit overzaelous. fixed :)

Reply:
The case handled here is handled later :).